### ISSUE TITLE
`poetry self:update` not working on Windows

### ISSUE THREAD

**Issue by moigagoo** [sdispater/poetry#132]
```
$ poetry self:update
Updating to 0.9.1

[RuntimeError]
Unable to determine poetry's path

self:update [--preview] [--] [<version>]
```

**Comment by moigagoo** [comment]
Proposed fix: https://github.com/sdispater/poetry/pull/135

**Comment by jgirardet** [comment]
samething on a ubuntu  16.04

**Comment by moigagoo** [comment]
Fixed in 0.10.3.
